setup.cfg

```diff
@@ -0,0 +1,95 @@
+[metadata]
+author = SciTools Developers
+author_email = scitools-iris-dev@googlegroups.com
+classifiers =
+    Development Status :: 5 Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Atmospheric Science
+    Topic :: Scientific/Engineering :: Visualization
+description =
+    A powerful, format-agnostic, community-driven Python package for analysing and visualising Earth science data
+download_url = https://github.com/SciTools/iris
+install_requires =
```

Comment:
I think `install_requires` should be in the `[options]` section

Reply:
Oops, yup, see https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html#options

Reply:
I thought the opposite! All the other packages in `[options]` are indeed optional, but there is no point installing Iris without the packages listed in `install_requires`. @jamesp can you explain your thinking for me? Thanks

Reply:
@trexfeathers not much thinking... it doesn't work if you put `install_requires` in `[metadata]`!

Reply:
i.e. if you install with `pip`, no dependencies will be installed if `install_requires` is not in `[options]`